.github/workflows/scan.yml

```diff
@@ -0,0 +1,55 @@
+---
+name: Scan
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+env:
+  IMAGE_NAME: opendatacube/geobase
+
+jobs:
+  cve-scanner:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout git
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Get unstable git tag
+        run: >
+          echo ::set-env name=UNSTABLE_TAG::$(git describe --tags)
+
+      - name: Log the unstable tag
+        run: echo $UNSTABLE_TAG
+
+      - name: Build unstable + latest Docker image tag
```

Comment:
I think we should pull from hub rather than build fresh.

Reply:
Also there is not one but 3 image tags, yes it's not ideal...

Reply:
Did you want me to scan all 3 images?

Reply:
I guess so, currently we only run `:runner`  derived images in k8s, but I think we should start running builder/wheels in k8s as well to compile our custom wheels

Reply:
I've added all 3 images, pulling from Docker Hub.